app/models/moab_storage_root.rb

```diff
@@ -25,8 +25,8 @@ def c2m_check!(last_checked_b4_date = Time.current)
 
   # Use a queue to ensure each druid on this root's directory is in the catalog database
   def m2c_check!
-    Stanford::MoabStorageDirectory.find_moab_paths(storage_location) do |druid, path, _match|
-      MoabToCatalogJob.perform_later(self, druid, path)
+    Stanford::MoabStorageDirectory.find_moab_paths(storage_location) do |druid, _match|
```

Comment:
Won't changing the block args to an arity of two assign the former value of `path` to `_match`? On line 28, I'd go with one of two changes:

1. If the block wants arity 3, re-add the `path` block arg and rename it `_path` (the leading underscore is a convention for unused variables);
2. If the block is OK with arity < 3, go with a single block arg for `druid`.

Looking good otherwise. Thanks for considering the changes!

Reply:
Just pushed update following option 1 that is consistent with usage at https://github.com/sul-dlss/preservation_catalog/blob/ff4cc9fc0f827b3966ecc7d82ee8f604ec868c14/spec/factories/preserved_object.rb#L18